core-bundle/contao/models/ContentModel.php

```diff
@@ -53,7 +53,7 @@
  * @property string            $highlight
  * @property string            $markdownSource
  * @property string|null       $code
- * @property string            $url
+ * @property string|null       $url
```

Comment:
Since this is a varchar field with empty string default, can it really be null or would it just be an empty string?

Reply:
@aschempp `tl_content.url` is `text NULL`, not a varchar.